apps/webstore/src/app/pages/auth/checkout/checkout.component.html

```diff
@@ -59,7 +59,7 @@
                         i18n="@@invalid-name"
```

Comment:
The localiztion attributes (`i18n`) have to be removed because otherwise they override the localized text returned by the validation method

Applies to every instance where it is present along the error message code